docs/conf.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/stable/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = u'MIS Builder'
+copyright = u'2018, Odoo Community Association (OCA)'
```

Comment:
do you need to add the license?

Reply:
CC BY license added done at https://github.com/OCA/mis-builder/commit/b6c6bf5e2700b6a115efa0d9f820f5fadfd36a14#diff-cac3dc3bfd42bc240bc9eb924f8dbfdd